src/image-viewer.js

```diff
@@ -0,0 +1,648 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
could you let me know the sections that have changed between this and the original?

Reply:
It's actually pretty cleanly in this single commit: https://github.com/ampproject/amphtml/pull/12092/commits/fd9837dc539f0914a4261f118d15c4c09cdcf787

I also marked them on the PR itself.